src/backends.jl

```diff
@@ -135,7 +135,7 @@ function pickDefaultBackend()
             Pkg.installed(env_default)  # this will error if not installed
             sym = Symbol(lowercase(env_default))
             if haskey(_backendType, sym)
-                return backend(sym)
+                return @eval backend($sym)
```

Comment:
Right, actually have read the warning :)
I've reverted the `@eval` stuff, since using `invokelatest` in Atom does indeed fix the worldage bug.

That also should make this PR fairly uncontroversial all in all now.